include/rmm/device_buffer.hpp

```diff
@@ -410,4 +392,22 @@ class device_buffer {
       mr::get_default_resource()};  ///< The memory resource used to
                                     ///< allocate/deallocate device memory
 };
+
+/**--------------------------------------------------------------------------*
+ * @brief Copies rmm::device_buffer to a preallocated host buffer.
+ *-------------------------------------------------------------------------**/
```

Comment:
Add a line to this doc of the form. "Copies device memory asynchronously on the specified stream."

Reply:
Thanks. Added.